.github/workflows/bmad-qa-gate-fast.yml

```
@@ -0,0 +1,71 @@
+        run: npx playwright install --with-deps chromium
+
+      - name: Run BMAD QA Gate
+        run: npm run qa:gate
+        env:
+          SUPABASE_PROJECT_ID: ${{ secrets.SUPABASE_PROJECT_ID }}
+          SUPABASE_URL: ${{ secrets.SUPABASE_URL }}
+          SUPABASE_ANON_KEY: ${{ secrets.SUPABASE_ANON_KEY }}
+          SUPABASE_SERVICE_ROLE_KEY: ${{ secrets.SUPABASE_SERVICE_ROLE_KEY }}
+          NODE_ENV: "test"
+          CI: "true"
+          RUNNER_TYPE: ${{ runner.labels[0] }}
```

actionlint (1.7.7)

property "labels" is not defined in object type {arch: string; debug: string; environment: string; name: string; os: string; temp: string; tool_cache: string}

(expression)